superbench/analyzer/data_analysis.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""A module for data analysis."""
+
+import pandas as pd
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+from superbench.common.utils import logger
+
+
+def statistic(raw_data_df):
+    """Get the statistics of the raw data, including count, mean, std, min, max, 25%, 50%, 75%.
+
+    Args:
+        raw_data_df (DataFrame): raw data
+
+    Returns:
+        DataFrame: data statistics
+    """
+    if len(raw_data_df) == 0:
+        return pd.DataFrame()
+    data_statistics_df = raw_data_df.describe()
+    statistics_error = []
+    for column in list(raw_data_df.columns):
+        if column not in list(data_statistics_df.columns) and not raw_data_df[column].isnull().all():
```

Comment:
Is the logical expression wrong? I think you want to confirm whether all values in the missing column is null? It should be
`and raw_data_df[column].isnull().all()`  not `and not raw_data_df[column].isnull().all()`
BTW, looks the UT can't test this code path.

Reply:
it's not logical expression wrong, I mean for the columns that may not be float type, and UT has tested for this

Reply:
Got it.